kolibri/plugins/perseus_viewer/buildPerseus.js

```diff
@@ -0,0 +1,119 @@
+const fs = require('fs');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const mkdirp = require('mkdirp');
+const rimraf = require('rimraf');
+const webpack = require('webpack');
+const buildConfig = require('./perseusBuildConfig');
+const extractPerseusMessages = require('./extractPerseusMessages');
+
+const currentCwd = process.cwd();
+
+const submodulesPath = path.resolve(__dirname, './submodules');
+
+rimraf.sync(submodulesPath);
+
+mkdirp.sync(submodulesPath);
+
+process.chdir(submodulesPath);
+
+console.log('Cloning the Learning Equality Perseus repo');
+
+execSync('git clone https://github.com/learningequality/perseus.git');
+
+console.log('Learning Equality Perseus repo successfully cloned');
+
+process.chdir('perseus');
+
+console.log('Fetching all perseus submodules');
+
+execSync('git submodule update --init --recursive');
+
+console.log('All perseus submodules fetched');
+
+console.log('Installing perseus dependencies');
+
+execSync('yarn install');
+
+console.log('Perseus dependencies installed');
+
+process.chdir(__dirname);
+
+const target = path.resolve(__dirname, './assets/dist');
+
+console.log('Clearing dist folder');
+
+rimraf.sync(target);
+
+mkdirp.sync(target);
+
+const cssPathRegex = /(url\(['"]?)([^"')]+)?(['"]?\),? ?)/g;
+
+const files = [
+  path.resolve(__dirname, './submodules/perseus/stylesheets/local-only/khan-exercise.css'),
+  path.resolve(__dirname, './submodules/perseus/lib/katex/katex.css'),
+  path.resolve(__dirname, './submodules/perseus/build/perseus.css'),
+  path.resolve(__dirname, './submodules/perseus/lib/mathquill/mathquill.css'),
+];
+
+const compiler = webpack(buildConfig);
+
+console.log('Creating Perseus bundle with webpack');
+
+const tokenStackRegex = /_token_stack:\n/g;
```

Comment:
a comment would be helpful - what is this?

Reply:
Oh - I can actually commit this change in the Perseus repo now I think about it.

It's a weird non-strict mode valid function label in the `KAS` jison-autogenerated parser module. It breaks building.